### DISCUSSION TITLE
多云自定义资源 - DaoCloud Enterprise

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1082] [Announcements]
# 多云自定义资源 - DaoCloud Enterprise

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/kairship/crds/crd/

**Comment by liyuerich** [comment]
返回 CR 实例列表页，即可查看刚刚创建的名为 my-new-cron-obiext 的 CR 实例。

应该是 my-new-cron-object

**Comment by windsonsea** [comment]
Fixed in pr #5268
